modules/jupyter-widget/src/widget.js

```diff
@@ -0,0 +1,112 @@
+import {DOMWidgetModel, DOMWidgetView} from '@jupyter-widgets/base';
+
+import {MODULE_NAME, MODULE_VERSION} from './version';
+
+import {createDeckScaffold, loadCss, hideMapboxCSSWarning, setMapProps} from './utils';
+
+const mapboxgl = require('mapbox-gl');
+const deckgl = require('@deck.gl/core');
+const deckglLayers = require('@deck.gl/layers');
+const deckAggregationLayers = require('@deck.gl/aggregation-layers');
+const deckJson = require('@deck.gl/json');
+
+const MAPBOX_CSS_URL = 'https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.1/mapbox-gl.css';
+const MAPBOX_TILE_URL = 'mapbox://styles/mapbox/light-v9';
+
+export class DeckGLModel extends DOMWidgetModel {
```

Comment:
Would love to see this class and methods more generously commented (explaining how a Jupyter Widget works).